app/models/pet.rb

```diff
@@ -32,7 +32,7 @@ class Pet < ApplicationRecord
 
   has_many :adopter_applications, dependent: :destroy
   has_many :tasks, dependent: :destroy
-  has_one :match, dependent: :destroy
+  has_many :matches, dependent: :destroy
```

Comment:
## Key change!!
Updated matches so that pets can have multiple matches, since they can be fostered multiple times as well as be adopted after being fostered.

All the tests pass, but I am worried there may be logic somewhere that is muddled by this. Is there anywhere that I missed that expected pets to only have one Match and would be broken now that they can have multiple?